requirements-test.txt

```diff
@@ -7,3 +7,5 @@ pytest-flakes==1.0.1
 pytest-mccabe==0.1
 pytest-pep8==1.0.6
 responses==0.5.0
+
+-e git+https://github.com/asher/chronos-python.git@e531f47d35d1d7b08d8e58abd58c3767af13f584#egg=chronos-python
```

Comment:
Ah, that's probably because it's still defined in `setup.py` here: https://github.com/shopkeep/shpkpr/blob/master/setup.py#L48

The requirements/dependency setup here is a bit convoluted, but basically the list in `setup.py` is used when installing the package from PyPI or Git (as happens during test or regular install), the list in `requirements.txt` is used when building the release docker container, and the list in `requirements-test.txt` is installed when spinning up a new test env.

I plan to simplify this over the next couple days once I merge my outstanding PRs. It works this way currently because we're doing dumb things with Docker and it was the only way to get caching to work at all acceptably. #78 should go some way to help fixing that.